risc0/zkvm/src/guest/env.rs

```diff
@@ -752,3 +752,17 @@ impl<F: Fn(&[u8])> std::io::Write for FdWriter<F> {
         Ok(())
     }
 }
+
+/// Read the input digest from the input commitment.
+pub fn input() -> Digest {
```

Comment:
In order to support a future version of this API that returns the preimage, we might want to call this `input_digest`. (If we don't rename this more generally)